eslint/eslint-patch/src/eslint-bulk-suppressions/generate-patched-file.ts

```diff
@@ -273,8 +273,19 @@ const requireFromPathToLinterJS = bulkSuppressionsPatch.requireFromPathToLinterJ
   }
 
   outputFile += `
-    // --- BEGIN MONKEY PATCH ---
-    if (bulkSuppressionsPatch.shouldBulkSuppress({ filename, currentNode: args[0]?.node ?? currentNode, ruleId, problem })) return;
+    // --- BEGIN MONKEY PATCH ---`;
+  if (majorVersion >= 9 && minorVersion >= 37) {
+    outputFile += `
+    if (bulkSuppressionsPatch.shouldBulkSuppress({ filename, currentNode: args[0]?.node ?? currentNode, ruleId, problem })) {
+      report.messages.splice(report.messages.indexOf(problem), 1);
```

Comment:
@iclanton , we should be moving these into `report.suppressedMessages` so that auditing can still find them. Also so that if we introduce the "unsuppressable rules" feature we can unsuppress them.